teamscale_client/client.py

```diff
@@ -334,6 +344,79 @@ def add_baseline(self, baseline):
         service_url += baseline.name
         return self.put(service_url, parameters={}, data=to_json(baseline))
 
+    def get_projects(self):
+        """Retrieves a list of projects from the server.
+
+        Returns:
+            List[:class:`data.ProjectInfo`]): The list of projects.
+
+        Raises:
+            ServiceError: If anything goes wrong
+        """
+        service_url = self.get_global_service_url("projects")
+        parameters = {
+            "detail": True
+        }
+        headers = {'Accept': 'application/json'}
+        response = requests.get(service_url, params=parameters, auth=self.auth_header, verify=self.sslverify,
```

Comment:
Use self.get()? This will spare you doing your own error handling and header prep.